GeneralsMD/Code/Libraries/Source/WWVegas/WWLib/rawfile.cpp

```diff
@@ -1110,22 +1032,8 @@ unsigned long RawFileClass::Get_Date_Time(void)
  *=============================================================================================*/
 bool RawFileClass::Set_Date_Time(unsigned long datetime)
 {
-#ifdef _UNIX
-	assert(0);
+	// TheSuperHackers @compile feliwir 08/04/2025 apparently this function is not called anyways
 	return(false);
```

Comment:
Put an DEBUG_ASSERTLOG with message for not being implemented. Comment can be removed then.